include/wil/wistd_functional.h

```diff
@@ -395,8 +401,7 @@ namespace wistd     // ("Windows Implementation" std)
 
     template<class _Rp, class ..._ArgTypes>
     function<_Rp(_ArgTypes...)>&
-    function<_Rp(_ArgTypes...)>::operator=(function&& __f)
-    {
+    function<_Rp(_ArgTypes...)>::operator=(function&& __f) WI_NOEXCEPT     {

```

Comment:
[style nit] A line break was lost so the { is on the wrong line and there is a bunch of spaces in between.